grammar.js

```diff
@@ -237,6 +238,15 @@ module.exports = grammar(C, {
       optional($._type_identifier)
     )),
 
+    variadic_declaration: $ => prec(1, seq(
```

Comment:
What do you think of calling this `variadic_parameter_declaration`?

Reply:
Works for me.

Reply:
Well, there's already a `variadic_parameter_declaration`, so that doesn't work...

Reply:
Turns out this new type isn't necessary.

Reply:
Oh, wonderful. I forgot about that.